### DISCUSSION TITLE
Actions is not triggering

### DISCUSSION THREAD

**Discussion by barnuri** [community/community#132107] [Actions]
### Select Topic Area

Bug

### Body

when the github actions doing releases or commits or pull request, from a github action its not triggering all other actions
example:
i have a github actions that creating a PR the pr owner is github action, and the action on PR like linting and unitests  not triggering, if i add an empty commit with my user to the branch everything works

**Comment by GreeningSiren** [comment]
Hello,
Could you please link the repository with which you are having problems

**Reply by barnuri** [reply]
its a private repo so i cant 
i can create a dummpy repo and reproduce the issue if you want

**Reply by GreeningSiren** [reply]
Can you just give us the github action .yml file?

**Reply by barnuri** [reply]
yes, the first one creating PR, the second one is running test on it
[ymls.zip](https://github.com/user-attachments/files/16243902/ymls.zip)

**Reply by GreeningSiren** [reply]
Sorry for replying so late, 
I was wondering if you got it fixed

If you have not, I think that the problem could be in the `on` property for the test action. It says ` on:  pull_request: branches: [master]` which could be wrong (idk). Try changing it to just `pull_request:` without the branches extension to see if it works

**Reply by barnuri** [reply]
hi, tried that, its not working too
its looks like not triggering because its from github actions bot user

**Reply by barnuri** [reply]
any update ?

**Comment by barnuri** [comment]
anyone can help ?

**Comment by jongpo-kim-otsk** [comment]
Can you check the your PR is create correctly in the Github UI?
And 
Can you check up Branch Rule Set about the master

**Reply by barnuri** [reply]
he created correctly, also the rule is ok
for example if i added empty commit to a PR that github action created, the commit checks on PR back to work

**Reply by barnuri** [reply]
?

**Comment by Tarian300** [comment]
Tried to use a workflow_run trigger in your workflow file?

**Reply by barnuri** [reply]
i dont want it to be manual

**Reply by nebuk89** [reply]
Does it work if you trigger it manually though? To check that everything else is good?
